### ISSUE TITLE
About gradient explosion in fine-tuning

### ISSUE THREAD

**Issue by Kaiwen-Zhu** [FoundationVision/Infinity#48]
Thanks for the excellent work! I want to ask about an issue I encountered during fine-tuning. I am trying to fine-tune for an img2img task, but in an attempt the gradients suddenly became very large during training, causing a sudden drop in model performance (see the curves below). Have you encountered a similar situation? Do you think this means there's something wrong with my current implementation? Thank you!
<details><summary>Details</summary>
<p>
Gradient norm

![Image](https://github.com/user-attachments/assets/89bb2d5b-0a79-44de-ac65-1c520465820f)

Acc bit
![Image](https://github.com/user-attachments/assets/d1ef4237-607d-4ce1-b48e-a3608fd47692)
</p>
</details>

**Comment by JeyesHan** [comment]
@Kaiwen-Zhu Your grad norm is extremely large. I think that may you miss some necessary normalizaiton when calculating loss or your learing rate is too large.

**Comment by Kaiwen-Zhu** [comment]
> [@Kaiwen-Zhu](https://github.com/Kaiwen-Zhu) Your grad norm is extremely large. I think that may you miss some necessary normalizaiton when calculating loss or your learing rate is too large.

@JeyesHan Thanks for your helpful advice
